app/controllers/api/v1/materials_controller.rb

```diff
@@ -221,6 +236,44 @@ def get_favorites
 
   end
 
+  #
+  #
+  # Get a single materials item and return a json representation
+  # GET /api/v1/materials/get_material
+  #
+  def get_material
```

Comment:
@scytacki Of course this breaks the API endpoint exposed to javascript through the Portal object:

<pre>
34:     MATERIALS_BIN_COLLECTIONS: "#{api_v1_materials_bin_collections_path}",
35:     MATERIALS_BIN_UNOFFICIAL_MATERIALS: "#{api_v1_materials_bin_unofficial_materials_path}",
36:     MATERIALS_BIN_UNOFFICIAL_MATERIALS_AUTHORS: "#{api_v1_materials_bin_unofficial_materials_authors_path}",
37:     <b>MATERIALS_GET_MATERIAL:     "#{api_v1_materials_get_material_path}",</b>
</pre>

Do any of our other API endpoints follow this pattern?